r2/r2/controllers/api.py

```diff
@@ -752,6 +754,9 @@ def POST_leavecontributor(self, container):
         """
         if container and container.is_contributor(c.user):
             container.remove_contributor(c.user)
+            if container.is_moderator_with_perms(c.user, 'access'):
```

Comment:
This seems unrelated, probably shouldn't be in here (or at least in a separate commit). I'm also not really sure why it's being added, why do we want a mod log entry when mods remove themselves as a contributor, when there isn't one for any other user removing themselves?